KsApi/extensions/NSURLSession.swift

```diff
@@ -23,7 +24,7 @@ internal extension URLSession {
   func rac_dataResponse(_ request: URLRequest, uploading file: (url: URL, name: String)? = nil,
                         and error: ErrorHandler? = nil)
     -> SignalProducer<Data, ErrorEnvelope> {
-    let producer = file.map { self.rac_dataWithRequest(request, uploading: $0, named: $1) }
+    let producer = file.map { self.rac_requestWithFileUpload(request, uploading: $0, named: $1) }
```

Comment:
Could use another eye here - pretty sure what this is doing originally was calling one method if `file` exists, and another if it doesn't, but it's confusing to me that it's via `file.map`.